packages/react-tabster/src/symbols.ts

```diff
@@ -0,0 +1,2 @@
+export const KEYBOARD_NAV_ATTRIBUTE = 'data-keyboard-nav' as const;
```

Comment:
What is this good for? It is not used anywhere.

Reply:
it's useless, I'll remove it